test/com/shashi/beans/OrderBeanTest.java

```diff
@@ -0,0 +1,91 @@
+package com.shashi.beans;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class OrderBeanTest {
+
+    @Test
+    public void testTransactionIdGetterSetter() {
+        OrderBean orderBean = new OrderBean();
+        String transactionId = "T123";
+        orderBean.setTransactionId(transactionId);
+        assertEquals(transactionId, orderBean.getTransactionId());
+    }
+
+    @Test
+    public void testProductIdGetterSetter() {
+        OrderBean orderBean = new OrderBean();
+        String productId = "P456";
+        orderBean.setProductId(productId);
+        assertEquals(productId, orderBean.getProductId());
+    }
+
+    @Test
+    public void testQuantityGetterSetter() {
+        OrderBean orderBean = new OrderBean();
+        int quantity = 5;
+        orderBean.setQuantity(quantity);
+        assertEquals(quantity, orderBean.getQuantity());
+    }
+
+    @Test
+    public void testAmountGetterSetter() {
+        OrderBean orderBean = new OrderBean();
+        double amount = 100.0;
+        orderBean.setAmount(amount);
+        assertEquals(amount, orderBean.getAmount(), 0.001); // Using delta for double comparison
+    }
+
+    @Test
+    public void testShippedGetterSetter() {
+        OrderBean orderBean = new OrderBean();
+        int shipped = 1;
+        orderBean.setShipped(shipped);
+        assertEquals(shipped, orderBean.getShipped());
+    }
+
+    @Test
+    public void testConstructorWithoutShipped() {
+        String transactionId = "T123";
+        String productId = "P456";
+        int quantity = 5;
+        double amount = 100.0;
+
+        OrderBean orderBean = new OrderBean(transactionId, productId, quantity, amount);
+
+        assertEquals(transactionId, orderBean.getTransactionId());
+        assertEquals(productId, orderBean.getProductId());
+        assertEquals(quantity, orderBean.getQuantity());
```

Comment:
this is wrong, please change it

Reply:
It is not necessary to do tests for beans as they do not have methods except getter and setter

Reply:
Focus should be more on service implementation classes